11ty/definitions/corporate-blackface.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Corporate Blackface
+slug: 'corporate-blackface'
+speech: noun
+defined: true
+excerpt: the idea that corporations publicly declare support of the Black Lives Matter movement, often with vague language, without actually committing to or implementing real, systemic change within their company to support Black employees, clients, customers, and community
+reading:
+  - text: You Can Keep The Corporate Blackface
+    href: https://www.tanyavital.com/news/keep-the-corporate-black-face
+  - text: Executives need to move beyond 'corporate blackface'
+    href: https://www.ft.com/content/77506820-deac-47b8-a44c-e588a5b8b92d
+
+---
+
+the idea that corporations publicly declare support of the Black Lives Matter movement, often with vague language, without actually committing to or implementing real, systemic change within their company to support Black employees, clients, customers, and community
+
+## Examples
+
+When a company:
+
+- releases statements, claiming to support Black lives, but fails to fix the toxicity and discrimination against Black people within their own company culture
+- openly showcases black people in its marketing materials, but doesn't actually include Black folks in any of its user research
+- as all-white leadership and tweets about its support of Black Lives Matter without addressing its complicity within white supremacy
+- does little to increase diversity and inclusion of Black voices in their company while stating that they are fighting for diversity and inclusion
+- claims to care about diversity and representation, but then fires their leading engineer, who also happens to be a woman of color, for calling them out on their racial bias
+- no longer actively tries to increase diversity and inclusion in their company after Black Lives Matter is no longer trending on social media
+
+There are so many ways that corporations show again and again that they only really care about their image, rather than Black lives.
```

Comment:
```suggestion
While the list above is finite, there are many more ways that corporations show again and again that they only really care about their image, rather than Black lives.
```